### ISSUE TITLE
faker.json does not work

### ISSUE THREAD

**Issue by cyril36** [joke2k/faker#1246]
* Faker version: Faker==4.1.1
* OS: Debian GNU/Linux 10 (buster)
I can not use faker.json as documented in the following link :  
https://faker.readthedocs.io/en/master/providers/faker.providers.misc.html#faker.providers.misc.Provider.json
such as:
`fake.json(data_columns=[('id', 'pyint', {'max_value':20})], num_rows=3)
`
I get the following error :   
AttributeError: 'Generator' object has no attribute 'json'

### Steps to reproduce

1. step 1
code : 
```
from django.test import TestCase
import factory
from faker import Faker
from faker.providers import misc
from upload.factories.fileUpload import FileUploadFactory
from django.core.files.uploadedfile import SimpleUploadedFile

fake=Faker()
fake.add_provider(misc)
 self.jpg_record = factory.django.FileField(from_path='upload/tests/image.jpg') #model.filefield
 self.good_metadata = fake.json(data_columns=[('id', 'pyint', {'max_value':20})], num_rows=3)#fake.json(data_columns=[('id', 'pyint'),('service','word')]) #model.jsonfield
instances = FileUploadFactory.create_batch(5,record=self.jpg_record,metadata=self.good_metadata)
for instance in instances:
            self.assertEqual("instance", instance,msg='{} , {} , {} , {}'.format(instance.pk,instance.owner,instance.metadata,instance.record))
```
    

1. step 2
execute : python3 manage.py test

### Expected behavior

should print my metadata field as a json object.

### Actual behavior
the faker module does not generate the json value and exit with the following error : 
```
Traceback (most recent call last):
  File "/usr/src/app/upload/tests/tests_models.py", line 26, in setUp
    self.good_metadata = fake.json(data_columns=[('id', 'pyint', {'max_value':20})], num_rows=3)#fake.json(data_columns=[('id', 'pyint'),('service','word')])
  File "/usr/local/lib/python3.8/site-packages/faker/proxy.py", line 96, in __getattr__
    return getattr(self._factories[0], attr)
AttributeError: 'Generator' object has no attribute 'json'
```

**Comment by johnbrandborg** [comment]
JSON is currently only available on the master branch.  pip install git+https://github.com/joke2k/faker

**Comment by johnbrandborg** [comment]
JSON is currently only available on the master branch. Faker 4.1.1 is a stable release that doesn't include JSON. You can install it with 'pip install git+https://github.com/joke2k/faker'

**Comment by fcurella** [comment]
I've just released v4.1.2, which includes the `json` generator. Thank you!
